core/examples/json/sortables.json

```diff
@@ -19,6 +19,14 @@
          "description": "This is a synthetic sortable that the server computes on the fly based on the spatial extent of the resource.  Resources without spatial extent have zero area.",
          "type": "number"
       },
+      "relevance": {
+         "title": "Relevance of the result",
+         "description": "This is a synthetic sortable that the server computes on the fly to assign a relevance to the result with respect to the query parameters specified in the original request",
```

Comment:
-> This is a synthetic sortable at the server's discretion that the server dynamically computes to determine relevance to the result with respect to the query parameters specified in the original request

Reply:
@tomkralidis are you asking to add "at the server's discretion"?  Why?  All sortables are at the server's discresion.  That is to say that the server decides what is a sortable or not and then create the respond accordingly.

Reply:
Yes, however sorting alphabetically or numerically is more predictable than a full text search.  I guess the main point here is that for relevance ranking, the actual algorithm to rank and provide a "score" is, as mentioned today on OAB, "magic".

Reply:
I do not think we need to clarify more.

Reply:
The "magic" is how the value of computed and that is not relevant here just that is computed and exposed as a sortable to allow the results to be ordered accoring to the relevance ranking.  Are you trying to say that the computation is at the discration of the server?  If yes, then I can reword a bit to convey that.

Reply:
Works for me, +1